api/routes/api_news.py

```diff
@@ -73,3 +82,54 @@ def specific_article(service: NewsService = Provide[Container.news_service]):
         logger.error(f"Error in POST /: {str(e)}")
         get_custom_error_response(e)
 
+
+
+@news_app.route("/get/recommendation", methods=["GET"])
+@jwt_required()
+@inject
+@handle_exceptions
+@logger.log_function_call()
+async def get_recommendations_endpoint(
+        service_recs: RecommendationService = Provide[Container.recommendation_service],
+        service_news: NewsService = Provide[Container.news_service],
+    ):
+    try:
+        user_email = get_jwt_identity()
```

Comment:
What if somebody tries to get here with invalid jwt token? Are we guaranteed that `user_email` will be filled with data?